cron/photos.py

```diff
@@ -54,6 +55,33 @@ def download_photo(user):
         # Some students don't have photos if they never went for picture day
     return None
 
+    def download_photo(url):

```

Comment:
you already have `download_photo` above so you'll need to call this something else.

Reply:
Isn't it fine because the method signature is different?

Reply:
in Java, yes. not in Python...

Reply:
lol